### PR TITLE
Removes leftover embedding_function name / doc references from chroma/atlas vectorstores

### PR DESCRIPTION
**Pull request by EvanOman**
# Removes leftover embedding_function name / doc references from chroma/atlas vectorstores

The Chroma and Atlas vector stores had been refactored to use a `Embeddings` instance rather than an `embedding_function`.

This PR cleans up a few variable names and comments which still used the old name `embedding_function` but used the new `Embeddings` object. This should make those sections of the code more readable.

## Who can review?

Anybody, but @dev2049 in particular

### CODE REVIEW

**Comment by dev2049** [comment]
this is a great cleanup but breaks backwards compatibility, so we'll want to think of some workaround for that

**Comment by EvanOman** [comment]
@dev2049 Are you thinking about people using the old `embedding_function` as a named argument?

I could check the `kwargs` to see if the `Embeddings` is being passed as an `embedding_function` argument, use it, and then print a deprecation warning

**Comment by dev2049** [comment]
sounds reasonable to me @EvanOman!

**Comment by EvanOman** [comment]
@dev2049 Done, let me know if there any other suggestions 👍

**Comment by EvanOman** [comment]
Hey @dev2049 , just pinging one more time on this, thanks! 👍

File: langchain/vectorstores/chroma.py [hunk 1]
```diff
@@ -87,12 +89,22 @@ def __init__(
                     )
             self._client = chromadb.Client(self._client_settings)
 
-        self._embedding_function = embedding_function
+        if embeddings is None and "embedding_function" in kwargs:
+            # Use kwarg embedding_function and print a deprecation warning
+            assert isinstance(kwargs["embedding_function"], Embeddings), \
```
Comment (hwchase17) [inline langchain/vectorstores/chroma.py:94]: to validate user input lets not use asserts but rather 
```
if not isinstance(kwargs["embedding_function"],...):
    raise ValueError
```

Comment (EvanOman) [reply langchain/vectorstores/chroma.py]: @hwchase17 Done! 👍

**Comment by EvanOman** [comment]
Looks like the Vercel bot is not happy, I'd address the issue but I don't have any access on Vercel to see what's up

**Comment by leo-gan** [comment]
@EvanOman Hi , could you, please, resolve the merging issues? After that ping me and I push this PR for the review. Thanks!
Seems, you need just commit some dummy change and Vercel would reset.

**Comment by efriis** [comment]
Closing because the PR wouldn't line up with the current directory structure of the library (would need to be in /libs/langchain/langchain instead of /langchain). Feel free to reopen against the current head if it's still relevant!

**Pull request closed without merging**